lib/cisco_node_utils/dns_domain.rb

```diff
@@ -59,8 +59,24 @@ def create
         config_set('dnsclient', 'domain_list',
                    state: '', name: @name)
       else
-        config_set('dnsclient', 'domain_list_vrf',
-                   state: '', name: @name, vrf: @vrf)
+        # On some platforms attempts to create a new domain-list results
+        # in the error. 'ERROR: Deletion of VRF test in progresswait
+        # for it to complete'.  We handle this by trying up to 500 times
+        # before giving up.
+        tries = 500
+        begin
+          config_set('dnsclient', 'domain_list_vrf',
+                     state: '', name: @name, vrf: @vrf)
+        rescue Cisco::CliError => e
+          if /ERROR: Deletion of VRF test in progress/.match(e.to_s)
```

Comment:
Is 'test' the name of the VRF? If so, should it be replaced with a `.*` or a `#{@vrf}`?